src/Components/Alert/Components/Filters/__tests__/Color.jest.tsx

```diff
@@ -0,0 +1,44 @@
+import { mount } from "enzyme"
+import {
+  AlertProvider,
+  useAlertContext,
+} from "Components/Alert/Hooks/useAlertContext"
+import { Color } from "Components/Alert/Components/Filters/Color"
+
+jest.mock("Utils/Hooks/useMatchMedia", () => ({
+  __internal__useMatchMedia: () => ({}),
+}))
```

Comment:
I don't think we need to mock `useMatchMedia` hook in this test. Let's remove it?

Reply:
I'll remove it with the other tests I'm working on.